test cases/unit/125 command list run_command/meson.build

```diff
@@ -0,0 +1,12 @@
+project('compiler_object_in_run_command', 'c')
```

Comment:
If you're going to create an entirely new test entry for this then avoid adding a language to it, as that adds a lot of additional time to the test runtime, and we aren't actually using a compiler for anything here.